packages/cli/src/commands/sandbox/sandbox_command.ts

```diff
@@ -94,8 +96,8 @@ export class SandboxCommand
     }
     const watchExclusions = args.exclude ?? [];
     const clientConfigWritePath = await getClientConfigPath(
-      args['out-dir'],
-      args.format
+      args['config-out-dir'],
+      args['config-format']
```

Comment:
can we add a test which asserts that these values are flowing correctly?

Reply:
Done in [70969c4](https://github.com/aws-amplify/amplify-backend/pull/643/commits/70969c45aace60246430ffb7058fd898ec13f419)